lib/jekyll/regenerator.rb

```diff
@@ -4,6 +4,7 @@ class Regenerator
 
     def initialize(site)
       @site = site
+      @disabled = nil
```

Comment:
Can we create a `private` `attr_accessor`? You may have to do this via:

```ruby
attr_accessor :disabled
private :disabled, :disabled=
```

and change all references of `@disabled` to `disabled` and `@disabled =` to `self.disabled =`